ui/app/common/pnc-client/resources/GroupBuildResource.js

```diff
@@ -63,23 +63,18 @@
           url: ENDPOINT + '/brew-push',
           successNotification: false
         },
-        queryByUser: {
-          method: 'GET',
-          isPaged: true,
-          url: ENDPOINT + '/?q=user.id==:userId',
```

Comment:
I removed this function because you can define RSQL queries when calling the query. If you hardcode it to the URL then It will cause issues when RSQL helper trying to modify it.

Also, there are some format fixed automatically when I save the file.